test/e2e/sidecar_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	"net/http"
 	"testing"
 
+	"github.com/sirupsen/logrus"
```

Comment:
should be in the second group. Is the linter working?

Reply:
I don't think it works for tests